test/index.test.js

```diff
@@ -491,6 +491,14 @@ test('AWS.setSDK function should mock a specific AWS module', function(t) {
     });
   });
 
+  t.test('Modules with multi-parameter constructors can be set for mocking', function(st) {
+	awsMock.setSDK('aws-sdk');
+	awsMock.mock('CloudFront.Signer', 'getSignedUrl');
+	var signer = new AWS.CloudFront.Signer('key-pair-id', 'private-key');
+	st.ok(signer);
+	st.end();
```

Comment:
The width of the indent seems to be unified with 2.

Reply:
Updated for whitespacing.